src/components/DateTimePickerNative/DateTimePickerNative.vue

```diff
@@ -0,0 +1,186 @@
+<!--
+  - @copyright Copyright (c) 2022 Julia Kirschenheuter <julia.kirschenheuter@nextcloud.com>
+  -
+  - @author Julia Kirschenheuter <julia.kirschenheuter@nextcloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<docs>
+> We're wrapping the awesome native browser datepicker as an input with some type of date e.g. https://html.spec.whatwg.org/multipage/input.html#local-date-and-time-state-(type=datetime-local)
+> Please check there for all the available options: https://html.spec.whatwg.org/multipage/input.html#
+> All available types are: 'date', 'datetime-local', 'month', 'time' and 'week'.
+
+### Example: datetime-local
+```vue
+<template>
+	<span>
+		<DateTimePickerNative
+			v-model="value"
+			:label="label"
+			type="datetime-local" />
+	</span>
+</template>
+<script>
+	export default {
+		data() {
+			return {
+				value: new Date(),
+				label: 'please select a new date'
+			}
+		},
+	}
+</script>
+```
+
+</docs>
+
+<template>
+	<div class="native-datetime-picker">
+		<label :class="{ 'hidden-visually': noVisibleLabel }"
+			:for="`native-datetime-picker_${label}`">{{ label }}</label>
+		<input :id="`native-datetime-picker_${label}`"
+			class="native-datetime-picker--input"
+			:type="type"
+			v-bind="$attrs"
+			:value="formattedValue"
+			:min="formattedMin"
+			:max="formattedMax"
+			v-on="listeners">
+	</div>
+</template>
+
+<script>
+
+const inputDateTypes = ['date', 'datetime-local', 'month', 'time', 'week']
+
+export default {
+	name: 'DateTimePickerNative',
```

Comment:
Needs the new prefix:
```suggestion
	name: 'NcDateTimePickerNative',
```
All file names and imports need to be adjusted accordingly.